modules/twitter/client.go

```diff
@@ -12,19 +17,37 @@ import (
 
 // Client represents the data required to connect to the Twitter API
 type Client struct {
-	apiBase     string
-	bearerToken string
-	count       int
-	screenName  string
+	apiBase    string
+	count      int
+	screenName string
+	httpClient *http.Client
 }
 
 // NewClient creates and returns a new Twitter client
 func NewClient(settings *Settings) *Client {
+	var httpClient *http.Client
```

Comment:
These several lines are duplicated verbatim between this file and `modules/twitterstats/client.go`.  I don't know where to put them so that they can be shared, but if anyone has suggestions I'll do it.